{"name": "Негативная проверка авторизации курьера с неверным заполнением обязательного поля \"login\"", "status": "passed", "description": "Запрос должен вернуть ошибку и правильный код ответа", "attachments": [{"name": "stdout", "source": "37b1321a-8452-4929-b646-f24e89143ebc-attachment.txt", "type": "text/plain"}], "start": 1721557343533, "stop": 1721557344040, "uuid": "8fe1985f-3b8e-4bf3-b0d7-fe47019c47cc", "historyId": "4be623606a2b5440c09298a9f565c961", "testCaseId": "4be623606a2b5440c09298a9f565c961", "fullName": "test_authorization_courier.test_authorization_with_wrong_login.TestAutorizationCourierWithWrongLogin#test_authorization_courier_with_wrong_login_error", "labels": [{"name": "tag", "value": "@pytest.mark.usefixtures('reg')"}, {"name": "parentSuite", "value": "test_authorization_courier"}, {"name": "suite", "value": "test_authorization_with_wrong_login"}, {"name": "subSuite", "value": "TestAutorizationCourierWithWrongLogin"}, {"name": "host", "value": "DESKTOP-IKUPUFD"}, {"name": "thread", "value": "4440-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_authorization_courier.test_authorization_with_wrong_login"}]}